crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -19,7 +19,11 @@ declare_rule! {
     /// Tooling also benefits from explicit file extensions, because they do not need to guess which
     /// file to resolve.
     ///
-    /// Rule checks static imports and dynamic imports calls such as `import()` and `require()`.
+    /// The rule checks static imports and dynamic imports calls such as `import()` and `require()`.
+    ///
+    /// To ensure that the TypeScript LSP adds the file extension when it automatically imports a variable,
+    /// you may set [`javascript.preferences.importModuleSpecifierEnding` and `typescript.preferences.importModuleSpecifierEnding`](https://code.visualstudio.com/updates/v1_24#_preferences-for-auto-imports-and-generated-code) to the desired file extension.
```

Comment:
Wrong link for `javascript.preferences.importModuleSpecifierEnding` and `typescript.preferences.importModuleSpecifierEnding`?